orchagent/vnetorch.cpp

```diff
@@ -898,9 +1062,56 @@ bool VNetBitmapObject::addRoute(IpPrefix& ipPrefix, nextHop& nh)
         throw std::runtime_error("VNet route creation failed");
     }
 
+    routeMap_.emplace(ipPrefix, routeInfo);
+
+    return true;
+}
+
+bool VNetBitmapObject::removeRoute(IpPrefix& ipPrefix)
+{
+    SWSS_LOG_ENTER();
+
+    if (routeMap_.find(ipPrefix) == routeMap_.end())
+    {
+        SWSS_LOG_WARN("VNET route %s doesn't exist", ipPrefix.to_string().c_str());
+        return false;
```

Comment:
if it doesn't exist, may be we can return `true `to avoid further retry.

Reply:
If it doesn't exist then something went wrong, so I believe we cannot just return ```true```. Also ```false``` is returned here to handle situation when ```add``` and ```remove``` operations are received in wrong order for the same entry.